storage/spider/spd_db_mysql.cc

```diff
@@ -4149,6 +4149,9 @@ int spider_db_mbase_util::open_item_func(
       last_str = SPIDER_SQL_IS_NOT_NULL_STR;
       last_str_length = SPIDER_SQL_IS_NOT_NULL_LEN;
       break;
+    case Item_func::JSON_EXTRACT_FUNC:
+      /* JSON_EXTRACT_FUNC should be processed the same way as UNKNOWN_FUNC */
+      /* fall through */
```

Comment:
We already know the Functype of `JSON_EXTRACT`, and thus there seems to be no need to handle it as if it is  `UNKNOWN_FUNC`. In fact, `JSON_EXTRACT` doesn't match any of if ... else if conditions under `CASE Item_func::UNKNOWN_FUNC`, and is processed by `spd_db_mysql.cc:4787-4800`.

It seems to be better, for me, to treat JSON_EXTRACT as other functions that have known types. In other words, I prefer to treat JSON_EXTRACT as the following:

```
      ...
    case Item_func::JSON_EXTRACT_FUNC:
      /* code only for JSON_EXTRACT */
      break;
    case Item_func::FooBar:
      ...
```